awpy/visualization/plot.py

```diff
@@ -431,58 +424,49 @@ def _plot_map_control_snapshot_helper(
             color_arr = ["#de9b35"] * len(ct_positions)
-            ax.scatter(transformed_x, transformed_y, c=color_arr)
-    ax.axis("off")
+            axes.scatter(transformed_x, transformed_y, c=color_arr)
+    axes.axis("off")
 
 
 def plot_map_control_snapshot(
     map_name: str,
-    tile_values: FrameMapControl,
-    player_pos: dict | None = None,
+    tile_values: FrameMapControlValues,
+    player_pos: FrameTeamMetadataObject | None = None,
     given_fig_ax: tuple[plt.Figure, plt.Axes] | tuple[None, None] = (None, None),
     save_filepath: str = "",
 ) -> None:
     """Visualize map control for given ct/t map control value dictionaries.
 
     Args:
-        map_name (string)       : Map used position_transform call
-        tile_values (dict)            : List with map control values dictionary
-                                  for both sides
-                                    Expected format = [CT Dict, T Dict]
-        player_pos (dict)       : Dictionary of player positions
-                                  for each team. Expected format
-                                  same as output of extract_player_positions
-        given_fig_ax            : Optional tuple containing figure and ax
-                                  objects for plotting
-        save_filepath (str)     : Filepath to save the figure to file
-                                  if required
+        map_name (str): Map used position_transform call
+        tile_values (dict): List with map control values dictionary for both sides
+            Expected format = [CT Dict, T Dict]
+        player_pos (dict): Dictionary of player positions for each team.
+            Expected format same as output of extract_player_positions
+        given_fig_ax: Optional tuple containing figure and ax objects for plotting
+        save_filepath (str) : Filepath to save the figure to file if required
 
     Returns: Nothing, all plotting is done on ax object
 
     Raises:
         ValueError: If map_name is not in awpy.data.NAV
     """
     if map_name not in NAV:
-        raise ValueError("Map not found.")
-    if "ct" not in tile_values.keys() or "t" not in tile_values.keys():
-        raise ValueError("Tile values argument not in expected FrameMapControl format")
-    ct_tiles, t_tiles = tile_values["ct"], tile_values["t"]
-    f, base_ax = given_fig_ax
-    if f is not None:
-        _plot_map_control_snapshot_helper(
-            map_name, ct_tiles, t_tiles, base_ax, player_data=player_pos
-        )
-    else:
-        f, base_ax = plot_map(map_name=map_name, map_type="simpleradar", dark=True)
-        _plot_map_control_snapshot_helper(
-            map_name, ct_tiles, t_tiles, base_ax, player_data=player_pos
-        )
+        msg = "Map not found."
+        raise ValueError(msg)
 
-    if len(save_filepath) > 0:
-        f.savefig(fname=save_filepath, bbox_inches="tight", dpi=400)
-        plt.close(f)
+    ct_tiles, t_tiles = tile_values["ct"], tile_values["t"]
```

Comment:
These probably dont need  to be assigned to variables.

Reply:
Yup addressed this in next commit to branch.